core-bundle/src/Cron/Cron.php

```diff
@@ -34,8 +36,27 @@ class Cron
      * @param \Closure():CronJobRepository      $repository
      * @param \Closure():EntityManagerInterface $entityManager
      */
-    public function __construct(private \Closure $repository, private \Closure $entityManager, private LoggerInterface|null $logger = null)
+    public function __construct(private \Closure $repository, private \Closure $entityManager, private CacheItemPoolInterface $cachePool, private LoggerInterface|null $logger = null)
     {
+        $this->addCronJob(new CronJob($this, '* * * * *', 'updateMinutelyCliCron'));
+    }
+
+    public function hasMinutelyCliCron(): bool
+    {
+        $item = $this->cachePool->getItem(self::MINUTELY_CACHE_KEY);
+
+        return $item->isHit();
+    }
+
+    public function updateMinutelyCliCron(string $scope): void
+    {
+        if (self::SCOPE_CLI !== $scope) {
+            return;
+        }
+
+        $cacheItem = $this->cachePool->getItem(self::MINUTELY_CACHE_KEY);
+        $cacheItem->expiresAfter(70); // 70 instead of 60 seconds to give some time for stale caches
```

Comment:
I’d even increase this to something like 300 (or 600), so that long running crons or other issues don’t accidentally enable the frontend cron.

Reply:
This would mean though that a cron like this:

```
*/5 * * * * vendor/bin/contao-console cron:run
```
or
```
```
*/10 * * * * vendor/bin/contao-console cron:run
```
respectively is still "allowed".

Reply:
Yes, in both cases no frontend cron should be needed, right?

Reply:
Well iirc @Toflar wanted to make a _minutely_ cron execution mandatory in order to ensure that the message queue gets processed properly.

Reply:
Not just that. It just makes no sense to configure anything different than `* * * * *` on CLI. Our cron framework delegates so one needs to have the lowest configuration possible in order to have the other ones get their desired execution interval.

Reply:
BTW: Long running crons wouldn't be an issue, they would run in the process of the previous minute so I think we should keep it as is. Imho this is no blocker, we can still increase or lower the time later on if there are valid reasons to do so.

Reply:
Yeah but within the process the minutely cronjob will run last and it is currently synchronous, which means it could be executed much later. May be also return a promise for that minutely cron?

Reply:
Can do that, if desired. No problem :)

Reply:
Implemented in https://github.com/contao/contao/pull/5591/commits/145edfa4504e7dd0651854fd76573413ce518dac.